docs/install-flox.md

```diff
@@ -120,7 +120,7 @@ description: Install or upgrade Flox
 
     1. Download installer for Macs with
 
-        [Apple M1/M2][flox_mac_m1_install]{:target="_blank" .md-button .md-button--primary}
+        [Apple M1/M2/M3][flox_mac_m1_install]{:target="_blank" .md-button .md-button--primary}
```

Comment:
I think I agree with Zach that using "Apple Silicon" is a nice way to avoid having to add the M4 whenever it's released. That said, this can totally be shipped as is.